doc/api/readline.md

```diff
@@ -225,97 +166,174 @@ rl.on('SIGTSTP', () => {
+  * `0` - the entire line
+
+The `readline.clearLine()` method clears current line of given [TTY][] stream
+in a specified direction identified by `dir`.
 
-* `-1` - to the left from cursor
-* `1` - to the right from cursor
-* `0` - the entire line
 
 ## readline.clearScreenDown(stream)
 
-Clears the screen from the current position of the cursor down.
+* `stream` {Writable}
+
+The `readline.clearScreenDown()` method clears the given [TTY][] stream from
+the current position of the cursor down.
 
 ## readline.createInterface(options)
 
-Creates a readline `Interface` instance. Accepts an `options` Object that takes
-the following values:
+* `options` {Object}
+  * `input` {Readable} The [Readable][] stream to listen to. This option is
+    *required*.
+  * `output` {Writable} The [Writable][] stream to write readline data to.
+  * `completer` {Function} An optional function used for Tab autocompletion.
+  * `terminal` {boolean} `true` if the `input` and `output` streams should be
+    treated like a TTY, and have ANSI/VT100 escape codes written to it.
+    Defaults to checking `isTTY` on the `output` stream upon instantiation.
+  * `historySize` {number} maximum number of history lines retained. To disable
+    the history set this value to `0`. Defaults to `30`. This option makes sense
+    only if `terminal` is set to `true` by the user or by an internal `output`
+    check, otherwise the history caching mechanism is not initialized at all.
 
- - `input` - the readable stream to listen to (Required).
+The `readline.createInterface()` method creates a new `readline.Interface`
+instance.
 
- - `output` - the writable stream to write readline data to (Optional).
+For example:
 
- - `completer` - an optional function that is used for Tab autocompletion. See
-   below for an example of using this.
+```js
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+```
 
- - `terminal` - pass `true` if the `input` and `output` streams should be
-   treated like a TTY, and have ANSI/VT100 escape codes written to it.
-   Defaults to checking `isTTY` on the `output` stream upon instantiation.
+Once the `readline.Interface` instance is created, the most common case is to
+listen for the `'line'` event:
 
- - `historySize` - maximum number of history lines retained. To disable the
-   history set this value to `0`. Defaults to `30`. This option makes sense
-   only if `terminal` is set to `true` by the user or by an internal `output`
-   check, otherwise the history caching mechanism is not initialized at all.
+```js
+rl.on('line', (line) => {
+  console.log(`Received: ${line}`);
+});
+```
 
-The `completer` function is given the current line entered by the user, and
-is supposed to return an Array with 2 entries:
+If `terminal` is `true` for this instance then the `output` stream will get
+the best compatibility if it defines an `output.columns` property, and emits
```

Comment:
The `,` here is unnecessary and, imho, confusing, because it takes a few parsing attempts to understand that `emits` refers to the output stream :)